### ISSUE TITLE
Add command schedule logging.

### ISSUE THREAD

**Issue by blu28** [ToughTechs151/robot-template#76]
We should note command scheduling transitions. There is an example in the WPI docs:
````
// Set the scheduler to log Shuffleboard events for command initialize, interrupt, finish
    CommandScheduler.getInstance()
        .onCommandInitialize(
            command ->
                Shuffleboard.addEventMarker(
                    "Command initialized", command.getName(), EventImportance.kNormal));
    CommandScheduler.getInstance()
        .onCommandInterrupt(
            command ->
                Shuffleboard.addEventMarker(
                    "Command interrupted", command.getName(), EventImportance.kNormal));
    CommandScheduler.getInstance()
        .onCommandFinish(
            command ->
                Shuffleboard.addEventMarker(
                    "Command finished", command.getName(), EventImportance.kNormal));
````
It would be best if this code were in Datalogging.java. Perhaps initialized automatically or by a separate method call?

**Comment by jwt388** [comment]
This code is already in dataLogging plus logging of each command execute which can be viewed in AutoScope console.  Additionally, command status of the scheduler and individual subsystems can be put to the dashboard.  To get useful information we need to add withName decorator to each command and change Arm commands from runOnce to startEnd so the last command stays active until the next one takes over.
![EventLog](https://github.com/ToughTechs151/robot-template/assets/151573770/0f4683cf-af57-45c5-a55d-0e59fafd81f5)
![CommandStatus](https://github.com/ToughTechs151/robot-template/assets/151573770/c96e4f2f-6fe7-4378-8c84-7546f354052f)

**Comment by jwt388** [comment]
Closed in commit 836569d6
